tagilmo/VereyaPython/timestamped_video_frame.py

```diff
@@ -82,4 +79,7 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.calibrationMatrix = np.reshape(np.asarray(loadedjson['projectionMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
         jo_len = jo_len + 4
         received_img_bytes = message.data[jo_len:]
-        self.pixels = cv2.cvtColor(cv2.imdecode(np.asarray(bytearray(received_img_bytes), dtype="uint8"), cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)
\ No newline at end of file
+        self._pixels = received_img_bytes
+
```

Comment:
```suggestion
    @property
```

Reply:
making it a property old code can read pixels like before, like pixels is a field

Reply:
@vsbogd i am not 100% sure about this, maybe it is better to let know api user that he or she calls a function here, what do you think?

Reply:
To me it is a matter of taste. Properties are definitely convenient for replacing a field by a method. If you can replace the field by a call in a client code you can just replace it. If you cannot (for instance you have a published API and cannot fix all clients) then the `@property` is the only way.